.github/workflows/acceptance.yml

```diff
@@ -0,0 +1,39 @@
+name: Acceptance
```

Comment:
FYI the GOPATH magic was found here: https://github.com/mvdan/github-actions-golang#how-do-i-set-up-a-gopath-build